main.goreleaser.yml

```diff
@@ -11,4 +11,12 @@ archives:
     format: tar.gz
     files:
       - test.sh
-      - custom_executable
+      - signature_printer
```

Comment:
Let's try to keep this to a single "directory" entry, and just place everything in that directory